service/src/test/java/uk/gov/mca/beacons/api/accountholder/application/AccountHolderServiceIntegrationTest.java

```diff
@@ -59,12 +62,15 @@ public void whenCreatingAccountHolder_ShouldPublishEvent() {
   @Test
   public void whenUpdatingAccountHolder_ShouldPublishEvent() {
     AccountHolder accountHolder = new AccountHolder();
-    accountHolder.setAuthId("test@test.com");
+    String testAuthId = "478879a5-03c7-42cd-a466-442ecf6dc2b7";
```

Comment:
We're relying on this user existing already. For a true test I feel that it should 

Setup 

1. Create a user. 
2. Get that user and assert it exists 

Update test
3. Assert user exists and name is x
4. Update user
5. Assert user still exists and has new name 

Cleanup
6. Remove test user
7. Assert user doesn't exist

Reply:
I'm not sure about three assertions in a single test but absolutely I'll set up and tear down the user instead

Reply:
Yeah I wouldn't say in a single test, but in a lifetime of the test, I think in the setup for all tests it could create a user to do stuff with, then at the end of the test delete the user. 

Then separate tests to make sure the creation and deletion works